specs/pipelines/run.spec

```diff
@@ -24,7 +24,7 @@ Steps:
       |1   |testdata/v1alpha1/pipelinerun/pipelinerun.yaml|
       |2   |testdata/v1beta1/pipelinerun/pipelinerun.yaml |
   * Verify pipelinerun
-      |S.NO|pipeline_run_name     |status     |check_lable_propagation|
+      |S.NO|pipeline_run_name     |status     |check_label_propagation|
```

Comment:
There are other files which is also having typo for `check_label_propagation` please include them :)

Reply:
I found a lot of such issues, I want to address them in new PR so that this one stays focused on just one thing - dependencies

Reply:
Ack